### ISSUE TITLE
SmartCity-Accessibility #210 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#143]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 65 lines...]
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 60 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/NearbyPlacesAttempt.java: Some input files use or override a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/NearbyPlacesAttempt.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[26,40] cannot find symbol
  symbol:   class LocationListCallback
  location: package smartcity.accessibility.database
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/database/LocationManager.java:[131,61] cannot find symbol
  symbol:   class LocationListCallback
  location: class smartcity.accessibility.database.LocationManager
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[155,62] cannot find symbol
  symbol:   class LocationListCallback
  location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[INFO] 3 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 14.784 s
[INFO] Finished at: 2017-01-09T11:45:28+00:00
[INFO] Final Memory: 27M/154M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[26,40] cannot find symbol
[ERROR] symbol:   class LocationListCallback
[ERROR] location: package smartcity.accessibility.database
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/database/LocationManager.java:[131,61] cannot find symbol
[ERROR] symbol:   class LocationListCallback
[ERROR] location: class smartcity.accessibility.database.LocationManager
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[155,62] cannot find symbol
[ERROR] symbol:   class LocationListCallback
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] a17692074b2f17f0876754aa687f187c89d9c0ed - added base for #141 and changed the right click function a bit

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/210/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
